### PR TITLE
feat(journey-maps): custom map attribution component

### PR DESCRIPTION
**Pull request by sebisteiner**
Refs: ROKAS-1259

### CODE REVIEW

**Review by sebisteiner — COMMENTED** [review]
The © icon looks different in Chrome and Firefox

**Review by swiss-chris — APPROVED** [review]
LGTM

File: src/journey-maps/angular/components/attribution/attribution.scss [hunk 1]
```diff
@@ -1,7 +1,84 @@
...
+    display: flex;
+    flex-wrap: wrap;
+    padding: 0 5px;
+    font-size: 12px;
+    align-items: center;
+
+    .attribution-item {
+      margin-right: 0.5em;
+
+      &.var-last {
```
[the remark this answers is not in the document]
Comment (sebisteiner) [reply src/journey-maps/angular/components/attribution/attribution.scss]: nice - didn't know `column-gap`.

File: src/journey-maps/angular/components/attribution/attribution.ts [hunk 2]
```diff
@@ -1,12 +1,118 @@
...
+  @Input() map: MaplibreMap | null;
 
-// Will be implemented with https://flow.sbb.ch/browse/ROKAS-90.
-// At the moment we only override some of the maplibre styles.
-export class SbbAttribution {}
+  open: boolean;
+  compact: boolean;
+  attributions: string[] = [];
+
+  private _listeners = new Map<string[], (event?: MapDataEvent) => void>();
```
[the remark this answers is not in the document]
Comment (sebisteiner) [reply src/journey-maps/angular/components/attribution/attribution.ts]: 👍

**Pull request merged**